src/main/webapp/student/scripts/play-game.js

```diff
@@ -242,5 +278,47 @@ function createAnswer(quiz, multipleDiv, doc, i){
       multipleDiv.appendChild(boxDiv);
 }
 
+async function showAnswers(){
+      if(currentQuestionActive){
+        document.getElementById("result").innerText = 'Wait for question to end...';
+      } else {
+        var tag;
+        firebase.auth().currentUser.getIdToken(/* forceRefresh */ true).then(async function(idToken) {
+        const infoJson = await fetch('/answer?gameInstance='+gameInstanceId+'&student='+idToken);
+        const info = await infoJson.json(); 
+        isCorrect = info.correct;
+        resultObject.classList.toggle(isCorrect.toString());
+        if(isCorrect) {
+            tag = "celebrate";
+            resultObject.innerText = 'Correct!';
+        } else {
+            tag = "disappointment";
+            resultObject.innerText = 'Incorrect :(';
+        }
```

Comment:
Nit: We can add the word "Disney" to the tag, so even if it has a hard time filtering some stuff, we know that it's really family friendly

Reply:
Right